hangman.c

```diff
@@ -0,0 +1,312 @@
+//Olivier Gabison - Hangman Game
+//NOTE: I don't understand the error here as well?
+
+#include <stdio.h>
+
+/*
+WORD TO GUESS: CHOCOLATE
+NUMBER OF MAXIMUM ERRORS: 5
+*/
+
+int main(){
+
+  int errors = 5;
+  char answer = {'c','h','o','c','o','l','a','t','e','\0'};
+  char guess = {'_','_','_','_','_','_','_','_','_','\0'};
+  char incorrect[50];
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your first guess.\n");
+  char guess1;
+  scanf("%c",&guess1);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1; // If x = 1, guess is incorrect. if x = 0, guess is right
+    if(guess1 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your second guess.\n");
+
+  char guess2;
+  scanf("%c",&guess2);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess2 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly");
+      errors--;
+      break;
+    }
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your third guess.\n");
+
+  char guess3;
+  scanf("%c",&guess3);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess3 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your fourth guess.\n");
+
+  char guess4;
+  scanf("%c",&guess4);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess4 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+  printIncorrectLetters();
```

Comment:
you can put breaks (though they're kind of useless). it has to do with how you handled methods!